manifests/d/DOSBoxStaging/DOSBoxStaging/0.80.1/DOSBoxStaging.DOSBoxStaging.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.3.4 $debug=NVS1.CRLF.5-1-19041-4123.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DOSBoxStaging.DOSBoxStaging
+PackageVersion: 0.80.1
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/dosbox-staging/dosbox-staging/releases/download/v0.80.1/dosbox-staging-v0.80.1-setup.exe
+  InstallerSha256: 94B961F397F3B70A13DAF3B573857B5262515E75D490BFC19F019479FE1694F8
+  UpgradeBehavior: install
+  InstallerSwitches:
+    Silent: /sp- /verysilent /norestart
+    SilentWithProgress: /sp- /silent /norestart
+- Architecture: x86
```

Comment:
Are you sure that the portable binary is 32-bit? I sadly cannot check it since I don't have any Windows machine here at the moment.